CONTRIBUTING.md

```diff
@@ -56,13 +56,32 @@ large_files:
     url: "url-of-model.ts"
 ```
 
+### Necessary Files and Keys
+
+In order to be compatible with other apps such as MONAI FL, MONAI deploy and MONAI Label, except the MONAI Bundle Specification, a bundle in MONAI Model Zoo should contain the following necessary files:
```

Comment:
I think we should describe "for the typical supervised segmentation and classification tasks...". Other tasks no need to follow the naming so far and maybe can't follow at all.
@wyli What do you think?

Thanks.

Reply:
HI @Nic-Ma , only these files are hard requirements:
```
1. `configs/metadata.json`
2. `models/model.pt` (or a download link in the config file for large files)
3. `configs/inference.json` (or `.yaml`, `.json`)
```
It seems these files are necessary.

Other requirements all have prerequisites (like if using preprocessing, it should be named as `preprocessing` exactly), and I think that using unify naming rules no matter which task the bundle belongs to is helpful if we have further supports on new tasks, for example, support GAN in MONAI deploy. What do you think?

Reply:
maybe `Necessary Files and Keys` should be `Preferred Files and Keys`, so that the bundle can have good interoperability with other tools such as MONAI FL, deploy SDK and Label...